packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -199,13 +199,21 @@ export const transformPlotlyJsonToDonutProps = (
 ): IDonutChartProps => {
   const firstData = input.data[0] as PieData;
 
-  const donutData = firstData.labels?.map((label: string, index: number): IChartDataPoint => {
+  const mapLegendToDataPoint: Record<string, IChartDataPoint> = {};
+  firstData.labels?.forEach((label: string, index: number) => {
     const color = getColor(label, colorMap, isDarkTheme);
-    return {
-      legend: label,
-      data: firstData.values?.[index] as number, //ToDo how to handle string data?
-      color,
-    };
+    //ToDo how to handle string data?
+    const value = typeof firstData.values?.[index] === 'number' ? (firstData.values[index] as number) : 1;
```

Comment:
in case value is 1 we should set the labels to percent instead of absolute.

Reply:
The values or percents are displayed based on the `textinfo` property, with the default being percent - similar to Plotly.
https://github.com/microsoft/fluentui/blob/16136745a3f61710863352d11e7999ffcef8bfb6/packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts#L241